angular-src/src/app/services/search.service.ts

```diff
@@ -121,8 +122,19 @@ export class SearchService {
         map( (response) => {
             return this._mapResults(response, returnType);
         })
-    );;
-      
-   }
+    );
+  }
+  public tableJoin(selectMoviesMeta: boolean, selectMoviesVote: boolean, selectMoviesCredits: boolean) {
```

Comment:
Nice work here! Perhaps there is a way to dynamically build the boolean list. For example, if we added another table that could possibly be joined, how much code would we have to alter?

Reply:
I was trying to do it that way, but I couldn’t get it working properly. I’ll focus on query 2.0 after this PR.